.changeset/two-suns-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+update the frameworks' cli versions used in C3
```

Comment:
could you please update this to what framework and from and to which versions did we update? when we automate this, let's also make the changesets this descriptive so that the changelog is actually useful. thanks